app/views/schools/add_participants/start_term.html.erb

```diff
@@ -15,7 +21,7 @@
         :name,
         legend: { text: title, tag: 'h1', size: 'xl' },
         caption: { text: @school.name, size: "xl" },
-        hint: { text: "This is when #{@wizard.full_name} will start their #{@wizard.ect_participant? ? 'induction and begin ECF-based training' : 'ECF-based mentor training'} at your school." }) %>
+        hint: { text: "This is when #{@wizard.full_name_or_you} will start #{@wizard.sit_mentor? ? 'your' : 'their'} #{@wizard.ect_participant? ? 'induction and begin ECF-based training' : 'ECF-based mentor training'} at your school." }) %>
```

Comment:
If possible, we'd use the contraction 'you’ll' here as it's easier for users to read. Is that a problem given that we may display the user's name instead? Thanks